python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -320,6 +320,7 @@ def setDefaults(self):
 
         # All sources should be good for PSF summary statistics.
         self.compute_summary_stats.starSelection = "calib_photometry_used"
+        self.compute_summary_stats.starSelector.flags.good = ["calib_photometry_used"]
```

Comment:
This is actually a bug in calibrateImage that I discussed with John P. yesterday that it doesn't propagate the psf used flag.  This will be fixed when that is fixed.  I'll add a to-do

Reply:
Ah, I see, thanks!